spec/address_book_spec.rb

```diff
@@ -55,35 +55,78 @@ def check_entry(entry, expected_name, expected_number, expected_email)
     it 'imports the 1st entry' do
       book.import_from_csv('entries.csv')
       entry_one = book.entries[0]
-      check_entry(entry_one, 'Bill', '555-555-4854', 'bill@blocmail.com')
+      check_entry(entry_one, 'Bill',
+                             '555-555-4854',
+                             'bill@blocmail.com')
     end
 
     it 'imports the 2nd entry' do
       book.import_from_csv('entries.csv')
       # Check the second entry
       entry_two = book.entries[1]
-      check_entry(entry_two, 'Bob', '555-555-5415', 'bob@blocmail.com')
+      check_entry(entry_two, 'Bob',
+                             '555-555-5415',
+                             'bob@blocmail.com')
     end
 
     it 'imports the 3rd entry' do
       book.import_from_csv('entries.csv')
       # Check the third entry
       entry_three = book.entries[2]
-      check_entry(entry_three, 'Joe', '555-555-3660', 'joe@blocmail.com')
+      check_entry(entry_three, 'Joe',
+                               '555-555-3660',
+                               'joe@blocmail.com')
     end
 
     it 'imports the 4th entry' do
       book.import_from_csv('entries.csv')
       # Check the fourth entry
       entry_four = book.entries[3]
-      check_entry(entry_four, 'Sally', '555-555-4646', 'sally@blocmail.com')
+      check_entry(entry_four, 'Sally',
+                              '555-555-4646',
+                              'sally@blocmail.com')
     end
 
     it 'imports the 5th entry' do
       book.import_from_csv('entries.csv')
       # Check the fifth entry
       entry_five = book.entries[4]
-      check_entry(entry_five, 'Sussie', '555-555-2036', 'sussie@blocmail.com')
+      check_entry(entry_five, 'Sussie',
+                              '555-555-2036',
+                              'sussie@blocmail.com')
+    end
+  end
+
+  context 'Import from entries_2.csv' do
+    it 'imports the correct number of entries from entries_2.csv' do
+      book.import_from_csv('entries_2.csv')
+      book_size = book.entries.size
+
+      expect(book_size).to eql 3
+    end
+
+    it 'imports the 1st entry from csv #2' do
+      book.import_from_csv('entries_2.csv')
+      entry_one = book.entries[0]
+      check_entry(entry_one, 'Gary',
+                             '000-000-0000',
+                             'gdawg@busey2016.com')
+    end
+
+    it 'imports the 2nd entry from csv #2' do
+      book.import_from_csv('entries_2.csv')
+      entry_two = book.entries[1]
+      check_entry(entry_two, 'Nate',
+                             '555-1-ALLTHENUMBERS',
+                             'nate@fictional.email')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.